test/README.md

```diff
@@ -109,27 +109,17 @@ make test
 
   * sanity tests
     ```
-    make sanity
-    ```
-
-  * extended tests
-    ```
-    make extended
+    make _sanity
```

Comment:
is there a reason not to continue to support the make targets without the '_' prefix?

Reply:
We still support target without '_', but would not recommend the use of it as it doesn't print java version and result summary.

Reply:
We should reverse that at some point (so the non _ version gives you the extras).

Reply:
Thanks. It's not a straight forward fix as we take advantage of make to add extra targets. We will think about this in the next update. We will modify the Jenkins configuration along with this PR.